Doc/library/imaplib.rst

```diff
@@ -30,12 +30,14 @@ Three classes are provided by the :mod:`imaplib` module, :class:`IMAP4` is the
 base class:
 
 
-.. class:: IMAP4(host='', port=IMAP4_PORT)
+.. class:: IMAP4(host='', port=IMAP4_PORT[, timeout=None])
```

Comment:
Can we remove the square brackets here? I don't think they are necessary here and doesn't seem to be consistent with other parameters with default values.